src/index.html

```diff
@@ -21,7 +24,420 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header page__header">
+      <a
+        href="index.html"
+        class="header__logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="header__logo-img"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__links">
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link is-active"
+            >
```

Comment:
Fix classes everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="index.html"
              class="nav__link is-active"
            >
```